DOCS-7862 Quality Gates Documentation Update (#23020)
DataDog/documentation: content/en/quality_gates/_index.md

<div class="alert alert-warning">Quality Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
{{< /site-region >}}

## Overview

Quality Gates allow you to control software quality by configuring rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.

> {{< img src="quality_gates/rule_type.png" alt="A pipeline rule that fails when code coverage for PCT is zero in Quality Gates" style="width:100%" >}}

Reviewer: I think this screenshot is not a good example. It should fail when the code coverage is "below or equal", not "above or equal"

Use Quality Gates to:

> * Create rules that block workflows by using data in Datadog, ensuring that only code that has passed tests affects your production environment.

Reviewer 2: ```suggestion
* Create rules that block workflows using data in Datadog, ensuring that only code that meets your standards end up in production.
```

* Give your organization the ability to decide what code makes it to production, enhancing your deployment discipline and mitigating potential issues in live environments.
* Continually improve code quality and system performance with precise enforcement and customizable rules.

> By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives. 

Reviewer 3: ```suggestion
By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives. 
```
This link takes me to /monitors/guide/github_gating/ -> is this correct ?

Author: Yes, this was originally a guide in the CI Visibility doc set that we migrated to the Monitors doc set!
